### ISSUE TITLE
could we make resolver URL configurable? currently it's hardcoded

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/frontend-astrooda#132]

**Comment by motame** [comment]
Yes, it can be made configurable via the config of the main mmoda module.
However we need to define a clear interface (input and output).

**Comment by volodymyrss** [comment]
Interface to the resolver API? Let's just keep the implementation we have. If we change the version we'll update it and make some description if needed.

For now, in this issue, the concern is just that the URL should be possible to change.

**Comment by motame** [comment]
Exactly the resolver API !
OK, let us put the priority on the update first.

**Comment by volodymyrss** [comment]
This is good to have but not not too urgent. It's not too complex hopefully.
